tests/functional/base_pages/submitted-response.page.js

```diff
@@ -0,0 +1,16 @@
+import BasePage from "./base.page";
+
+class SummaryResponsePage extends BasePage {
+  url() {
+    return `/submitted/${this.pageName}`;
+  }
+
+  summary() {
+    return `.summary`;
+  }
+
+  summaryRowState(sectionId) {
+    return `[data-qa="${sectionId}"]`;
```

Comment:
In the functional test a section id is not passed to this method, but it works as it returns any element that matches the `data-qa` attribute. The methods for accessing summary elements should be auto-generated from the schema using the same code as the final summary.

Reply:
Darn, should have stuck to the card and not wrote that functional test.